hc/accounts/tests/test_profile.py

```diff
@@ -32,16 +33,18 @@ def test_it_sends_report(self):
     def test_it_adds_team_member(self):
         self.client.login(username="alice@example.org", password="password")
 
-        form = {"invite_team_member": "1", "email": "frank@example.org"}
-        r = self.client.post("/accounts/profile/", form)
-        assert r.status_code == 200
+        check = Check(name="Alice", user=self.alice)
+        check.save()
+
+        form = {"invite_team_member": "1", "email": "frank@example.org", "check":"Alice"}
+        response = self.client.post("/accounts/profile/", form)
+        self.assertEqual(response.status_code, 200)
 
         member_emails = set()
         for member in self.alice.profile.member_set.all():
             member_emails.add(member.user.email)
 
-        ### Assert the existence of the member emails
-
+        # Assert the existence of the member emails
         self.assertTrue("frank@example.org" in member_emails)
 
         ###Assert that the email was sent and check email content
```

Comment:
Well commented code... However, you could also add the test that the email has been sent and checking the email content.

Reply:
Assertions to check if invites have been sent added